content/md/administration/catalog-volume-monitoring_fr.md

```diff
@@ -0,0 +1,73 @@
+---
+id: catalog-volume-monitoring
+themes: administration, akeneo-concepts, catalog-settings 
+title: Suivre la **taille de votre catalogue** (since 2.3)
+popular: false
+related: what-is-a-product-value, what-is-an-attribute, what-is-a-family, what-is-a-channel, what-is-a-locale, what-is-a-category
+---
+
+# Un écran dédié
+
+Un écran est disponible dans le PIM afin de suivre la taille du catalogue sur plusieurs axes.
+
+Cet écran vous donne par exemple le nombre de familles ou le nombre moyen d'attributs par famille.
+
+![Catalog volume monitoring dashboard](../img/Activity_CatalogVolumeMonitoring_fr.png)
+
+::: warning
+Cet écran n'est disponible que depuis la version 2.3 du PIM.
+:::
+
+# Où trouver cet écran ?
+
+Voici les étapes afin d'accéder à ce superbe (;)) écran:
+
+1. Cliquer sur le menu `Activité`
+1. Cliquer sur le sous-menu `Volumétrie du catalogue`
+1. Et voilà !
+
+# Comment lire ce tableau de bord ?
+
+## Les différentes sections
+
+L'écran `Volumétrie du catalogue` est composé de plusieurs sections.
+
+### Le nombre de valeurs de produit : l'information la plus importante
+
+La première valeur affichée représente le nombre total de [valeurs de produit](/articles/fr/what-is-a-product-value.html) contenues dans le PIM.
+C'est un très bon indicateur de la véritable valeur de votre PIM.
+
+![Widget du nombre de valeurs de produit](../img/Activity_CatalogVolumeMonitoring_PvWidget_fr.png)
+
+Nous vous encourageons à surveiller cette valeur régulièremeent. Plus le nombre de valeurs de produit est important, plus votre PIM vous est utile !
```

Comment:
Nope, it means that if you enriched a lot of data, you will have more product values, so that means that you really need a PIM (otherwise you'll still be enriching in multiple spreadsheets and have all the problems we know).